modules/core/src/utils/shader.js

```diff
@@ -30,6 +30,16 @@ export function mergeShaders(target, source) {
   }
   if ('modules' in source) {
     result.modules = (target.modules || []).concat(source.modules);
+
+    // Hack: prject32 and project64 cannot co-exist
+    if (source.modules.some(module => module === 'project64' || module.name === 'project64')) {
+      const index = result.modules.findIndex(
+        module => module === 'project32' || module.name === 'project32'
+      );
+      if (index >= 0) {
+        result.modules.splice(index, 1);
```

Comment:
Nit: log.warn to fix the source.

Reply:
This is going to happen with every layer that uses this extension. What are we warning the user about?

Reply:
I see, add that information to the comment ?